solr/core/src/test/org/apache/solr/search/facet/TestJsonFacets.java

```diff
@@ -3226,6 +3269,26 @@ private static String str(AVLTreeDigest digest) {
     return sb.toString();
   }
 
+  private static boolean doesFacetMethodSupportMinCountZero() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportNumericFields() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportTermsPrefix() {
+    return FacetField.FacetMethod.DEFAULT_METHOD != FacetField.FacetMethod.DVSTRING;
+  }
+
+  private static boolean doesFacetMethodSupportSortedSetDocValues() {
```

Comment:
Hmm, it should work.. dvstring supports multi-valued string fields at least (using SortedSetDocValues)

Reply:
It was failing on `DocValues.getSorted`, but maybe it's something about `sf.multiValued()` not being set for some reason.  I can have a closer look to see if I can figure it out.